### ISSUE TITLE
Canno tinstall

### ISSUE THREAD

**Issue by gewahl** [molly/static-timeline-generator#8]
Just went by the instructions, but cannot install. With "npm install" get countless warnings about old lockfile and errors.

**Comment by Andre601** [comment]
I encounter the same issues here.
During `npm install` am I getting a lot of warnings - most are just about deprecated versions - but also errors and similar.

Here's a full log: https://hastebin.com/ihanerimut.coffeescript

From what I get is it related to python? For once I can see the mention of python2 (Why is it using python2?!) and then about some syntax error?

Also, my Anti-virus reports and deletes a "potentially malicious file". And before you say anything, I trust this anti-virus more than I trust myself, and I think an npm install can and should work without these security warnings popping up.

Either way, I hope some help can be provided here, as I actually would like to use this software.

**Comment by molly** [comment]
@gewahl @Andre601 give it a shot now? I've brought the deps back up to date, which resolves the errors (at least when I build it).

**Comment by Andre601** [comment]
> give it a shot now? I've brought the deps back up to date, which resolves the errors (at least when I build it).

Yeah. Sadly went with another solution, especially since it doesn't rely on a single JS file to be modified (No offense, but that's awful design imo), so I don't have aneed for this anymore.

Tho, still thanks for the effort.
